CHANGELOG.md

```diff
@@ -21,6 +21,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
```

Comment:
Add `## [15.7.0]`?

Reply:
We add actual version number after successful publish of the package and push it to develop pipeline